### ISSUE TITLE
Entraînement sur 2021 avec XGBoost

### ISSUE THREAD

**Issue by danielaDaliesio** [linhnguyen0132/GeoLifeCLEF-2021_projet-IFT3710#3]
Entraîner le modèle XGBoost sur les données de 2021 en prenant en note les métriques de top-30 error ainsi que top-1 accuracy. De plus,
prendre en note les train/val loss pour voir l'évolution du modèle.
